src/env.cc

```diff
@@ -917,6 +917,17 @@ void AsyncHooks::grow_async_ids_stack() {
 uv_key_t Environment::thread_local_env = {};
 
 void Environment::Exit(int exit_code) {
+  if (options()->trace_exit) {
+    HandleScope handle_scope(isolate());
+
+    fprintf(stderr,
+            "(node:%d) WARNING: Exited the environment with code %d\n",
+            uv_os_getpid(),
+            exit_code);
+    PrintStackTrace(
+        isolate(),
+        StackTrace::CurrentStackTrace(isolate(), 10, StackTrace::kDetailed));
```

Comment:
We usually do

```suggestion
        StackTrace::CurrentStackTrace(isolate(), 10, StackTrace::kDetailed));
  fflush(stderr);
```

because in `node_main.cc` we disable stdio buffering iiuc.

Reply:
If the buffers were disabled, how come would `fflush` help? IIUC it just flushes the buffered data.

Reply:
Yeah, no idea. Maybe we can just remove all of them.